docs/app/pages/css/css-sections/icons/rotate-flip-icons/snippets/sample.html

```diff
@@ -1,7 +1,7 @@
-<p><i class="hpe-icon hpe-tab-up " aria-hidden="true"></i> no rotation</p>
-<p><i class="hpe-icon hpe-tab-up hpe-rotate-90" aria-hidden="true"></i>&nbsp;hpe-rotate-90</p>
-<p><i class="hpe-icon hpe-tab-up hpe-rotate-180" aria-hidden="true"></i>&nbsp;hpe-rotate-180</p>
-<p><i class="hpe-icon hpe-tab-up hpe-rotate-270" aria-hidden="true"></i>&nbsp;hpe-rotate-270</p>
-<p><i class="hpe-icon hpe-iteration" aria-hidden="true"></i>&nbsp;no flip</p>
-<p><i class="hpe-icon hpe-iteration hpe-flip-vertical" aria-hidden="true"></i>&nbsp;hpe-flip-vertical</p>
-<p><i class="hpe-icon hpe-iteration hpe-flip-horizontal" aria-hidden="true"></i>&nbsp;hpe-flip-horizontal</p>
\ No newline at end of file
+<p><i class="ux-icon ux-icon-tab-up " aria-hidden="true"></i> no rotation</p>
```

Comment:
I always thought it was a bit confusing to have the "no rotation" text here; could you remove this and "no flip" to emphasize that no class is necessary in these cases.

Reply:
Removed these examples, thanks